{{cookiecutter.project_name}}/Makefile

```diff
@@ -38,6 +35,11 @@ else
 {%- endif %}
 endif
 
+# Project paths
+PACKAGES := $(PACKAGE) tests
+CONFIG := $(shell python -c 'import glob; print("\n".join(glob.glob("*.py")))')
+MODULES := $(shell python -c 'import glob; print("\n".join(glob.glob("$(PACKAGE)/*.py")))')
```

Comment:
This does not search recursively, but serves the purpose of determining when to regenerate documentation.